spec/controllers/idv/in_person/verify_info_controller_spec.rb

```diff
@@ -171,6 +178,31 @@
         expect(@analytics).to have_logged_event('IdV: proofing resolution result missing')
       end
     end
+
+    context 'when idv/in_person data is present' do
+      before do
+        subject.user_session['idv/in_person'] = flow_session
+      end
+
+      it 'renders the show template without errors' do
+        get :show
+
+        expect(response).to render_template :show
+      end
+
+    end
+
+    context 'when idv/in_person data is missing' do
+      before do
+        subject.user_session['idv/in_person'] = {}
+      end
+
+      it 'redirects to idv_path' do
+        get :show
+        # expect(response.status).to eq 302
```

Comment:
Did you mean to leave this comment here?

Reply:
Ah good catch thank you, I'll clean this up!